interpreter/cling/test/Prompt/ValuePrinter/SourceLocation.C

```diff
@@ -8,22 +8,27 @@
 //------------------------------------------------------------------------------
 // RUN: cat %s | %cling | FileCheck %s
 
+.rawInput 1
 #include <iostream>
 #if __cplusplus >= 202002L
 #include <version>
 #endif
 
-#ifndef __cpp_lib_source_location
-// Hack to prevent failure if __cpp_lib_source_location feature does not exist!
-std::cout << "(std::source_location) ";
-std::cout << "CHECK_SRCLOC:42:std::source_location getsrcloc()\n";
-#else
+#ifdef __cpp_lib_source_location
 #include <source_location>
 std::source_location getsrcloc() {
 #line 42 "CHECK_SRCLOC"
   return std::source_location::current();
 }
-getsrcloc()
+#else
+// Hack to prevent failure if __cpp_lib_source_location feature does not exist!
+void getsrcloc() {
+  std::cout << "(std::source_location) ";
+  std::cout << "CHECK_SRCLOC:42:std::source_location getsrcloc()\n";
+}
 #endif
+.rawInput 0
```

Comment:
Just for curiosity: We need to always wrap it into a function because we need `.rawInput 0` before printing the return value?